test/support/helpers/cozy.js

```diff
@@ -1,6 +1,7 @@
 /* @flow */
 /* eslint-env mocha */
 
+require('isomorphic-fetch')
```

Comment:
Out of curiosity, how was it failing before adding this?

Reply:
It wouldn't find `fetch`.